### PR TITLE
doc(changelog): add information about the removal of RetrofitError

### PR DESCRIPTION
**Pull request by dbyron-sf**

### CODE REVIEW

**Review by xibz — COMMENTED** [review]
Overall looks good

> code that still does throw RetrofitError

while true, I think we need to reword this to just relying on retrofit errors, since users may be catching them in plugins, etc

**Comment by dbyron-sf** [comment]
> Overall looks good
> 
> > code that still does throw RetrofitError
> 
> while true, I think we need to reword this to just relying on retrofit errors, since users may be catching them in plugins, etc

Is https://github.com/spinnaker/spinnaker.io/pull/440/commits/2bee40e5dbaea5e057183b9bab88a577a5441360 better?

**Review by xibz — APPROVED** [review]
Thanks for the update! Looks good 🚢
